src/components/pages/photos/PhotosPage.js

```diff
@@ -5,10 +5,27 @@ import './PhotoPage.css';
 
 
 export default class PhotosPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentRover: 'Curiosity',
```

Comment:
Do we want the `currentRover` to default to a rover or not?

Should we display nothing until someone has selected a rover?

I'm not sure one way or the other but I think it's worth some discussion!